packages/components/src/components/post-collapsible/post-collapsible.tsx

```diff
@@ -69,8 +67,9 @@ export class PostCollapsible {
   async toggle(open = !this.isOpen): Promise<boolean> {
     if (open === this.isOpen) return open;
 
-    this.isOpen = !this.isOpen;
-    if (this.isLoaded) this.postToggle.emit(this.isOpen);
+    this.isOpen = open;
```

Comment:
Is this.isOpen still needed?

Reply:
Yes, this is still needed. The component initially renders the host without any styles (i.e., in the 'opened' state), even when it loads with the `collapsed="true"` prop. In this case, `this.isOpen === !this.collapsed` is only true after the toggle method has been called once in `componentDidLoad`.